.github/workflows/format.yml

```diff
@@ -0,0 +1,20 @@
+name: Format Golang
+
+on:
+  - pull_request
+
+jobs:
+    format_golang:
+      name: Format Golang
+      runs-on: ubuntu-latest
+      steps:
+        - name: Setup Golang
+          uses: actions/setup-go@v3
+          with:
+            go-version: 1.17
```

Comment:
The `go.mod` file in this repo defined `1.18`, so I guess the actions can run with that as well.

Reply:
Resolved suggestion:

Bumped to Go version 1.18